packer-qemu.json

```diff
@@ -1,5 +1,6 @@
 {
     "variables": {
+        "cpu"           : "qemu64",
```

Comment:
should this be `"host"` here too?

Reply:
This section sets the default. The values in the individual vars.json files override the default.  I _think_ "qemu64" is the CPU that's used if it's not otherwise specified, so I want to make that the default so the EL 7 and 8 builds don't change.